rest-api/test/unit_test/unit_test_util.py

```diff
@@ -177,8 +177,12 @@ def setup(self, with_data=True, with_views=False):
         # Default no-pw login, according to https://circleci.com/docs/1.0/manually/#databases .
         mysql_login = 'ubuntu'
       else:
+        if 'MYSQL_ROOT_PASSWORD' in os.environ:
```

Comment:
You can instead just do:

password = os.getenv('MYSQL_ROOT_PASSWORD', 'root')

Reply:
Done.